system_metrics_collector/src/system_metrics_collector/main.cpp

```diff
@@ -53,9 +54,17 @@ int main(int argc, char ** argv)
     STATISTICS_TOPIC_NAME,
     std::chrono::milliseconds(1000 * 60));
 
+  auto process_mem_node =
+    std::make_shared<system_metrics_collector::LinuxProcessMemoryMeasurementNode>(
+    "linuxProcessMemoryCollector",
+    std::chrono::milliseconds(1000),
```

Comment:
```suggestion
    1000ms,
```
https://en.cppreference.com/w/cpp/chrono/operator%22%22ms